cmd/lncli/arg_parse.go

```diff
@@ -38,3 +39,10 @@ func parseTime(s string, base time.Time) (uint64, error) {
 
 	return strconv.ParseUint(s, 10, 64)
 }
+
+const lightningPrefix = "lightning:"
+
+// stripPrefix removes prefix accidentally copied from the CLI.
+func stripPrefix(s string) string {
+	return strings.TrimSpace(strings.TrimPrefix(s, lightningPrefix))
```

Comment:
@yyforyongyu I couldn't figure out when the prefix is getting appended, so I wasn't sure about whitespaces. Do you think we need to trim? Or if u could give me a hint how to reproduce it. I'm bringing this up because it may be causing DeadlineExceeded https://app.travis-ci.com/github/lightningnetwork/lnd/jobs/588347497#L513 (it's just a guess)

Reply:
hmm i don't think that failure is related to this change. In fact, I don't think our `lncli` is covered in our integration tests, so they are unrelated.

> I couldn't figure out when the prefix is getting appended, so I wasn't sure about whitespace.

Not sure what you mean. I think the unit tests have covered the whitespace case?

Reply:
I wasn't sure if there's any whitespaces between the prefix and the invoice, that's the reason I've added strings.TrimSpace